source/retryable-writes/tests/README.rst

```diff
@@ -396,20 +396,25 @@ and sharded clusters.
    9. Disable the failpoint.
 
 #. Test that drivers return the original error after encountering a
-   WriteConcernError error with a RetryableWriteError label. This test MUST be
-   implemented by any driver that implements the Command Monitoring
-   specification. This prose test requires drivers to set a fail point after an
+   WriteConcernError with a RetryableWriteError label. This test MUST
+
+   1. be implemented by any driver that implements the Command Monitoring
+      specification,
+
+   2. only be run against replica sets as mongos does not propagate the
+      RetryableWriteError label to the drivers.
```

Comment:
Pedantically, mongos does propagate the RetryableWriteError label when the mongos itself is shutting down. So an InterruptedAtShutdown from the mongos itself shutting down should have the RetryableWriteError label attach but no instances of retryable errors returned by the shards to the mongos will have an error label attach if the error is propagated all the way back to the driver.

s/RetryableWriteError/NoWritesPerformed/ would allow you to omit these specifics.